ntc_templates/templates/huawei_vrp_display_vlan.textfsm

```diff
@@ -36,7 +44,13 @@ VLANS
 
 
 STATUS
```

Comment:
We could drop this extra blank line before `STATUS` state too.
Thank you @evilmonkey19

```suggestion

STATUS
```